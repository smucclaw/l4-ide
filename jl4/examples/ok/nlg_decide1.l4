ยง Example

ASSUME Person IS A TYPE
ASSUME walks  IS A BOOLEAN
ASSUME drinks IS A BOOLEAN
ASSUME eats   IS A BOOLEAN

GIVEN p [person] IS A Person
DECIDE `must sing`
    IF        walks [%p% must walk]
       AND    eats [%p% must eat]
           OR drinks
