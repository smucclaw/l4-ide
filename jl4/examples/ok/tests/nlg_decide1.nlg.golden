[8:7-8] p
  [8:9-17] [person]
[10:15-20] walks
  [10:21-43] [the person must walk]
[11:15-19] eats
  [11:20-41] [the person must eat]
